Kickstarter-iOS/Views/Controllers/ProjectPamphletViewControllerTests.swift

```diff
@@ -122,6 +122,38 @@ internal final class ProjectPamphletViewControllerTests: TestCase {
     }
   }
 
+  func testLoggedIn_Backer_LiveProject_Error_NativeCheckout_Enabled() {
+    let config = Config.template
+      |> \.features .~ [Feature.checkout.rawValue: true]
+    let currentUser = User.template
+    let backing = Backing.template
+      |> Backing.lens.status .~ .errored
+    let backedProject = Project.cosmicSurgery
+      |> Project.lens.photo.full .~ ""
+      |> Project.lens.personalization.isBacking .~ true
+      |> Project.lens.personalization.backing .~ backing
+      |> Project.lens.state .~ .live
+
+    [Device.phone4inch, Device.phone5_5inch, Device.phone5_8inch].forEach { device in
```

Comment:
Can we use `combos(Language.allLanguages, Device.allCases)` so it's consistent with the rest of the tests in this file? I know it generates tons of screenshots, but this is a high-value piece of UI that would be useful to have fully covered in all languages and screens sizes